lib/measured/unit.rb

```diff
@@ -53,7 +54,7 @@ def inverse_conversion_amount
 
   private
 
-  def with_case_sensitivity(comparison)
+  def with_case_sensitivity(case_sensitive, comparison)
```

Comment:
This method signature is super confusing. Why do we have to pass `case_sensitive` into `with_case_sensitivity`?